### PR TITLE
Task #54848 [LiconKA] Function-7 : Share

### PR DESCRIPTION
**Pull request by liconrepo**

### CODE REVIEW

**Comment by phamxuanlu** [comment]
OK

File: app/src/main/java/com/licon/rssfeeds/ui/activity/RssBaseDetailsActivity.java [hunk 1]
```diff
@@ -94,6 +96,18 @@ private void setFeedDataOnUI(FeedItem feedItem) {
         mCollapsingToolbarLayout.setTitle(!TextUtils.isEmpty(category) ? category : "");
 
         setLink(feedItem.getLink());
+
+        // crating news to share it...
+        newsBuilder.append(title)
```
Comment (nvmanh) [inline app/src/main/java/com/licon/rssfeeds/ui/activity/RssBaseDetailsActivity.java]: create method for this

File: app/src/main/java/com/licon/rssfeeds/ui/activity/RssBaseDetailsActivity.java [hunk 2]
```diff
@@ -94,6 +96,18 @@ private void setFeedDataOnUI(FeedItem feedItem) {
...
+                .append("\n")
+                .append(published_on)
+                .append("\n\n")
+                .append(description)
+                .append("\n")
+                .append(media_url)
+                .append("\n\n")
+                .append("Link: ").append(getLink())
```
Comment (nvmanh) [inline app/src/main/java/com/licon/rssfeeds/ui/activity/RssBaseDetailsActivity.java]: constant values

**Comment by nvmanh** [comment]
OK!

**Pull request merged**